src/metaschema/oscal_metadata_metaschema.xml

```diff
@@ -258,14 +311,13 @@
             <formal-name>Party Type</formal-name>
             <description>A category describing the kind of party the object describes.</description>
             <constraint>
-                <allowed-values allow-other="no">
```

Comment:
For consideration, are non-person users from system-to-system accounts or automation services deployed within an `party[@type='organization']` with a `member-of-organization` linkage constitute a `party[@type='person']` or do we need a separate type of enum?

Key Model Review Criteria:
- Is this proposed change to 1) the model itself, 2) documentation embedded in the model, or 3) both? 3
- Is this proposed change to 1) enhance the model in short-term way that will not break backwards compatibility, 2) a significant change that will require breaking backwards compatibility or 3) it is unclear, and I need to provide explanation? 1, we can potentially add an `enum/@value` if deemed necessary or allow-other='yes'.
- Is this proposed change to 1) current operational use of one model or 2) cross-document mechanics? 2 I think it will change mechanics.